app/models/refine/inline/criteria/input.rb

```diff
@@ -29,8 +29,8 @@ def attributes
       value: value,
       value1: value1,
       value2: value2,
-      count_refinement_attributes: count_refinement_attributes.presence,
-      date_refinement_attributes: date_refinement_attributes.presence
+      count_refinement: count_refinement_attributes.presence,
+      date_refinement: date_refinement_attributes.presence
```

Comment:
Definitely more for my own understanding and learning, do these key names need to match the `result[:input]` from the `/criterion.rb` below?

I see that the `#to_blueprint_node` has been simplified and would now include some of those key/values that match key names for `:count_refinement_attributes` and `:date_refinement_attributes`. Or do these two portions of code not really connect in that way?

Reply:
Good call out. In all honesty this very thing was why this PR took so long. I think they were named mainly this was as a translation step. This inline code is a new UI wrapper around the older code. `count_refinement` and `date_refinement` are standard expected condition criteria for the core gem behavior. The Inline UI create a bunch of Input classes in ruby that specify attributes, of which are included count_refinement_attributes, etc. So my understanding is this was simply to keep those two things separate even if the same data values are being passed between. So its not **necessarily** going to match the result[:input] but in most cases it will